runtime/vm/jvminit.c

```diff
@@ -1824,7 +1824,9 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				uint64_t limit = 0;
 				uint32_t rc = omrsysinfo_get_limit(OMRPORT_RESOURCE_FILE_DESCRIPTORS | J9PORT_LIMIT_HARD, &limit);
 				if (OMRPORT_LIMIT_UNKNOWN != rc ) {
-					omrsysinfo_set_limit(OMRPORT_RESOURCE_FILE_DESCRIPTORS | OMRPORT_LIMIT_SOFT, limit);
+					Trc_VM_SetSoftFileLimit(vm->mainThread, limit);
+					rc = omrsysinfo_set_limit(OMRPORT_RESOURCE_FILE_DESCRIPTORS | OMRPORT_LIMIT_SOFT, limit);
+					Trc_VM_SetSoftFileLimitResult(vm->mainThread, rc);
 				}
```

Comment:
Should there be another tracepoint in the case where omrsysinfo_get_limit fails (OMRPORT_LIMIT_UNKNOWN )?